include/zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20-enga.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
nit: adjust commit message for edf118179c7ac31a975dff996a8d77f85802f7b6 (now all FICR fields are there, not only HSFLL)